data/assets/scene/digitaluniverse/alternatestarlabels.asset

```diff
@@ -23,7 +23,7 @@ local Object = {
     Unit = "pc"
   },
   GUI = {
-    Name = "Stars Labels - Alternate",
+    Name = "Alternate Stars Labels",
```

Comment:
I'm also planning to possibly overhaul these alternate labels in the future. So, the whole star names situation may change at some point.

Reply:
Right now I put it in the `base.asset`. Have not committed those changes yet, tho. If it's in the DU asset will it load anywhere? `digitaluniverse.asset` seems to be a subset that is not used. Plus, I also set some ordering in the Solar System.

Reply:
How about making the digitaluniverse.asset load the same assets as the base and then do the reordering in there?  It doesn't seem very "base-y" to me to also muck with the order of the scene graph nodes and that asset it starting to do a lot

Reply:
I do not much care *where* this happens, but that it happens and is basically the default on any generic launch of OS. A few reasons for this:

One, seeing the data sets all jumbled in the scene list is off-putting. It is akin to reading a book on C and randomly, or alphabetically, ordering the chapters, then the sections within each chapter. It denies the natural progression that the material exists within. It is far more elegant to adhere to that natural order and endow this list with a sensible order. Yes, it could demand more curation as new sets are added, but those curation steps will bring profound meaning to the user, rather than a haphazard, meaningless order.

Two, I've structured the entire content chapter on this ordering in the guide. The whole progression of these sections reflects the natural progression of these data sets.

I'm open to implementation options... you just tell me how such that it proliferates throughout the OS ecosystem, generally.

Reply:
:+1:  Yea, I think in that case we can have your ordering code in the `digitaluniverse.asset` instead and then `asset.require` the digitaluniverse asset from the `base.asset` instead.  Right now the `base.asset` has its own list of DU assets to include and I think it would be better to just replace that list with a single `require` and fix the digitaluniverse asset instead